distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -703,54 +703,6 @@ public void testFederatedRetrieveProductInvalidResourceUrl() throws Exception {
         .body(containsString("Unable to transform Metacard."));
   }
 
-  /**
```

Comment:
The two tests removed from this class no longer make sense. They fail early because of permissions that protect the system and there's no reason for the tests to continue.